README.md

```diff
@@ -148,10 +168,6 @@ make -j 32 && make install
 
 Let's run `ctest` now on a compute node:
 
-### On Cori
```

Comment:
This was already fixed, make sure you rebase with the new develop and resolve all conflcts.

Reply:
already fixed on which branch? This PR was created right after I synced the code change from develop branch.

Reply:
Let's have a zoom chat some time today or tomorrow to go through this... Maybe there is something I didn't notice?

Reply:
The stable branch, can you rebase based on that?